release-content/0.16/migration-guides/_guides.toml

```
@@ -1030,3 +1030,9 @@ title = "Use `target_abi = \"sim\"` instead of `ios_simulator` feature"
 prs = [17702]
 areas = []
 file_name = "17702_Use_target_abi__sim_instead_of_ios_simulator_feature.md"
+
+[[guides]]
+title = "Improved Derive(Event)"
+prs = [18083]
+areas = ["Ecs"]
```

```suggestion
areas = ["ECS"]
```

The ECS area is fully capitalized